src/me/ryanhamshire/PopulationDensity/EntityEventHandler.java

```diff
@@ -126,14 +139,14 @@ public void onItemDespawn (ItemDespawnEvent event)
 		ItemStack item = ((Item)entity).getItemStack();
 		
 		//only care about saplings
-		if(item.getType() != Material.SAPLING) return;
+		if(saplings.contains(item.getType())) return;
```

Comment:
As aforementioned, standard for comparing enums is `==`

Reply:
is not an enum

Reply:
Whoops, read it too fast lol.